### PR TITLE
Add package liboggz version 1.1.1

### PR DESCRIPTION
**Pull request by mfrw**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
- Add package liboggz version 1.1.1
- liboggz: lint spec
- liboggz: cgmanifest: add entry
- liboggz: license: add entry

###### Change Log  

- liboggz: lint spec
- liboggz: verify license
- liboggz: cgmanifest: add entry
- liboggz: license: add entry

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

- Local Build

BuddyBuild:
- [AMD](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=291465&view=results)
- [ARM](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=291467&view=results)

### CODE REVIEW

File: SPECS-EXTENDED/liboggz/liboggz.spec [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+%files devel
+%{_includedir}/oggz
+%{_libdir}/liboggz.so
+%{_libdir}/pkgconfig/oggz.pc
+
+%files doc
+%doc __docs_staging/*
+
+%changelog
+* Wed Jan 02 2023 Muhammad Falak <mwani@microsoft.com> - 1.1.1-20
```
Comment (mfrw) [inline SPECS-EXTENDED/liboggz/liboggz.spec:104]:
```suggestion
* Mon Jan 02 2023 Muhammad Falak <mwani@microsoft.com> - 1.1.1-20
```

File: SPECS-EXTENDED/liboggz/liboggz.spec [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+Oggz provides a simple programming interface for reading and writing
+Ogg files and streams. Ogg is an interleaving data container developed
+by Monty at Xiph.Org, originally to support the Ogg Vorbis audio
+format.
+
+This package contains HTML documentation needed for development using
+liboggz.
+
+%prep
+%setup -q -n %{name}-%{version}
```
Comment (Sumynwa) [inline SPECS-EXTENDED/liboggz/liboggz.spec:51]: Consider using %autosetup

File: SPECS-EXTENDED/liboggz/liboggz.spec [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+%patch0 -p1 -b .multilib
+
+%build
+%configure --disable-static
+sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
+sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
+make %{?_smp_mflags}
```
Comment (Sumynwa) [inline SPECS-EXTENDED/liboggz/liboggz.spec:58]: Consider using %make_build

File: SPECS-EXTENDED/liboggz/liboggz.spec [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+
+%check
+# Tests disabled for moment because of rpath issue
+#make check
+
+%install
+rm -rf $RPM_BUILD_ROOT
```
Comment (Sumynwa) [inline SPECS-EXTENDED/liboggz/liboggz.spec:66]: Consider using %{buildroot}

File: SPECS-EXTENDED/liboggz/liboggz.spec [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+find %{buildroot} -type f -name "*.la" -delete -print
+
+# not particularly interested in the tex docs, the html version has everything
+rm -rf __docs_staging/latex
+
+# Multilib fix: ensure generated headers have timestamps
+# independent of build time
+(cd include/oggz &&
+    touch -r oggz_off_t_generated.h.in.multilib \
+      $RPM_BUILD_ROOT%{_includedir}/oggz/oggz_off_t_generated.h
```
Comment (Sumynwa) [inline SPECS-EXTENDED/liboggz/liboggz.spec:79]: Consider using %{buildroot}

File: SPECS-EXTENDED/liboggz/liboggz.spec [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+
+
+%ldconfig_scriptlets
+
+
+%files
+%license COPYING
+%doc AUTHORS ChangeLog README
+# 0 length NEWS file
+# %doc NEWS
```
Comment (Sumynwa) [inline SPECS-EXTENDED/liboggz/liboggz.spec:90]: Better to remove commented files check.

**Review by Sumynwa — APPROVED** [review]
Looks good. Just some minor comments. Thanks.

File: SPECS-EXTENDED/liboggz/liboggz.spec (hunk 2 above)
Comment (mfrw) [reply SPECS-EXTENDED/liboggz/liboggz.spec:51]: it seems the `%patch -b .multilib` is needed for the build.

**Pull request merged**